Duplicati/Library/Main/Operation/CompactHandler.cs

```diff
@@ -236,13 +236,18 @@ where report.CompactableVolumes.Contains(v.Name)
                             }
                         }
                     }
-                    
+                    else
+                    {
+                        newvolindex.Dispose();
+                        newvol.Dispose();

```

Comment:
This removes files that remained after a compact operation was run, but `report.ShouldCompact` is `false`.